Translations/Language.zh-cn.xml

```diff
@@ -1300,7 +1300,7 @@
     <entry lang="zh-cn" key="FILESYS_REPAIR_CONFIRM_BACKUP">警告：使用微软的 'chkdsk' 工具修复受损的文件系统时可能会导致受损区域的数据丢失。因此，建议您首先备份存储在 VeraCrypt 加密卷中的文件到另外完好的加密卷。\n\n您确认现在就修复文件系统吗？</entry>
     <entry lang="zh-cn" key="MOUNTED_CONTAINER_FORCED_READ_ONLY">由于写入权限被拒绝，加密卷 '%s' 不得不以只读模式加载。\n\n请确认文件类型加密卷的安全许可允许您对其执行写入操作（右键单击此容器并选择 属性 -&gt; 安全）。\n\n注意，由于 Windows 的原因，您可能在设置成合适的安全属性后还会看到这个警告，这并非是 VeraCrypt 的程序故障。一个可能的解决方法是移动您的加密卷文件，例如移动到文档文件夹。\n\n如果您确实要保持加密卷的只读模式，请设置加密卷的属性为只读（右键单击加密盘文件并选择属性 -&gt; 只读），这样就可以禁止显示警告了。</entry>
     <entry lang="zh-cn" key="MOUNTED_DEVICE_FORCED_READ_ONLY">由于写入权限被拒绝，加密卷 '%s' 不得不以只读模式加载。\n\n请确认没有其它程序（例如，防病毒程序）正在访问加密卷所在的分区/设备。</entry>
-    <entry lang="zh-cn" key="MOUNTED_DEVICE_FORCED_READ_ONLY_WRITE_PROTECTION">加密卷 '%s' 已使用只读模式装载，这是因为操作系统报告主设备为写保护状态。\n\n请注意，在一些第三方的芯片驱动中，已有错误报告指出可能会造成可写入介质错误的被系统报告为写保护。这个问题并不是由 VeraCrypt 造成。这可以通过更新或卸载当前系统中的任何第三方芯片组驱动（来自非微软官方）来尝试解决。</entry>
+    <entry lang="zh-cn" key="MOUNTED_DEVICE_FORCED_READ_ONLY_WRITE_PROTECTION">加密卷 '%s' 已使用只读模式加载，这是因为操作系统报告主设备为写保护状态。\n\n请注意，在一些第三方的芯片驱动中，已有错误报告指出可能会造成可写入介质错误的被系统报告为写保护。这个问题并不是由 VeraCrypt 造成。这可以通过更新或卸载当前系统中的任何第三方芯片组驱动（来自非微软官方）来尝试解决。</entry>
```

Comment:
I checked translation tools and they seems to agree that the term "装载" (mounted) would be more appropriate than "加载" (loaded):

- 装载 (Zhuāngzài): Often means "load", "mount", or "charge". It can be used in the context of loading goods, but it is also often used in technical terms for "mounting" (as in mounting a volume or drive).
- 加载 (Jiāzài): This typically means "load" or "loading", more commonly used in terms of software loading, data loading, or page loading.

For the specific context of "mounting a volume or drive", "装载" (from original translation) is more contextually accurate. While "加载" is more general for "loading", "装载" is closer to the specific action of mounting in VeraCrypt.

Can you please explain your point of view?

Reply:
“装载”一般用于物体，把物体装上；
“加载”通常用于数据，把数据加载到内存。

Reply:
I don't agree with the content of this PR. There was originally a meaningful distinction between "mount" (“装载/挂载”) and "load" (“加载”). In the Chinese context, "load" is usually translated as "加载," while "mount" corresponds to "装载" or "挂载." As @idrassi already found that:
> I checked translation tools and they seems to agree that the term "装载" (mounted) would be more appropriate than "加载" (loaded):

I understand that you changed it to "加载" because the word "load" is more commonly seen in daily life. However, please note that the reason why there are fewer translations for "装载" in the Chinese context is because there are fewer "mounts" in daily software operations. If such changes are made, it will destroy the original meaning of the words and cause broader confusion. It's important not to translate randomly based on personal preferences alone.

Specifically, in the VeraCrypt, the term “装载” has been in use for a long time, and making large-scale changes is bound to cause confusion for users.

Moreover, as proof that "mount" does not usually translate into “加载”, I directly searched for the keyword "mount 加载" using search engines. It's evident that search engines recognize this input as incorrect and search for the correct translation “装载/挂载.” This clearly shows that the translation of "装载" corresponds more accurately to the term "mount."